doc/changes/latest.rst

```diff
@@ -9,6 +9,7 @@
 NEW
 ---
 
+:func:`~mass_univariate.permuted_ols` and :func:`~glm.second_level.non_parametric_inference` now support :term:`TFCE`-based FWE correction (:gh:`3196` by `Taylor Salo`_).
```

Comment:
```suggestion
:func:`~mass_univariate.permuted_ols` and :func:`~glm.second_level.non_parametric_inference` now support :term:`TFCE` statistic (:gh:`3196` by `Taylor Salo`_).
```
(TFCE is a statistic, and the correction scheme is a standard non-parametric, aka pemutations).